sdk/src/types/block/output/chain_id.rs

```diff
@@ -3,18 +3,20 @@
 
 use derive_more::From;
 
-use crate::types::block::output::{AccountId, FoundryId, NftId, OutputId};
+use crate::types::block::output::{AccountId, DelegationId, FoundryId, NftId, OutputId};
 
 ///
 #[derive(Clone, Copy, Eq, Hash, PartialEq, Ord, PartialOrd, From)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub enum ChainId {
-    ///
+    /// An account id.
     Account(AccountId),
-    ///
+    /// A foundry id.
     Foundry(FoundryId),
-    ///
+    /// An NFT id.
     Nft(NftId),
+    /// A delegation id.
+    Delegation(DelegationId),
```

Comment:
while your at it ... add docs for `ChainId` as well?

Reply:
Not in this PR no, this should be done in develop

Reply:
Oh you meant just the struct itself, I guess I can yeah

Reply:
But the others should have been added to develop, maybe I'll just remove them and not add any doc here

Reply:
Removed

Reply:
alright